src/components/VoterStatus.js

```diff
@@ -111,6 +111,7 @@ const VoterStatus = ({
   signaling,
   onboardingState
 }) => {
+  console.log('voter status component, fetching?', fetching);
```

Comment:
@jparklev I'm kind of stumped on this. When calling `free()`, after the tx is mined, this VoterStatus component re-renders to display the updated balances. When calling `freeAll()`, after the tx is mined, VoterStatus does not get re-rendered, and the new balances are not displayed. I think there is a dispatch tied to it somewhere, but I can't seem to find the thread. I'll keep looking into it, but wondered if you had any thoughts?

Reply:
ah, i believe those changes are happening here
https://github.com/makerdao/governance-dashboard/blob/d1d365d5c3fbb0f9d1e0f11e0b26616712ceb9e4/src/reducers/accounts.js#L538
the `accounts` reducer is listening for the `WITHDRAW_MKR_SUCCESS` action and updating it's own store in response

Reply:
ah, thanks! That was it, keen eye.